src/languageSupport/languages/flux/lsp/connection.ts

```diff
@@ -319,7 +319,9 @@ export class ConnectionManager {
       previousState
     )
 
-    if (this._first_load) {
+    const multipleItemsToSync =
+      Object.keys(toAdd).length + Object.keys(toRemove).length > 1
```

Comment:
I love, love, love turning this expression into a variable.

Same suggestion as in the previous PR about boolean names (unless you decided you're not into that, in which case, ignore this).

```ts
const hasMultipleItemsToSync =
  Object.keys(toAdd).length + Object.keys(toRemove).length > 1
```

Reply:
done